packages/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp

```diff
@@ -71,8 +78,12 @@ std::shared_ptr<TurboModule> cxxModuleProvider(
   //   return std::make_shared<NativeCxxModuleExample>(jsInvoker);
   // }
 
+#if AUTOLINKING_AVAILABLE
   // And we fallback to the CXX module providers autolinked
   return autolinking_cxxModuleProvider(name, jsInvoker);
+#endif
+
+  return nullptr;
```

Comment:
Clang might complain about unreachable code (with `-Wall`):
```suggestion
#else
  return nullptr;
#endif
```